scripts/install-build-tools.sh

```diff
@@ -14,11 +14,41 @@ if (( $EUID != 0 )); then
     SUDO='sudo'
 fi
 
+# Function to check if a version of Python >= 3.10 is installed
+check_python_version() {
+  if command -v python3 &>/dev/null; then
+    # Get the version of Python
+    PYTHON_VERSION=$(python3 --version | awk '{print $2}')
+    IFS='.' read -r -a version_parts <<< "$PYTHON_VERSION"
+    echo "Python version: ${version_parts[0]}.${version_parts[1]}"
+    # >= 3.10
+    if (( ${version_parts[0]} >= 3 && ${version_parts[1]} >= 10 )); then
```

Comment:
Is there a reason to prefer arithmetic expansion `((`/`))` instead of plain tests?

Nit: There's also a lot of machinery (reading into an array variable from another variable assigned from a command substitution) which I think could/should be simplified.

Reply:
Added a simpler one-liner with grep and regex instead, thanks